BasicRPG/BasicRPG/Program.cs

```diff
@@ -1,160 +1,121 @@
-﻿using System.Runtime.CompilerServices;
-using System.Security.Cryptography.X509Certificates;
-
-class Program
+﻿class Program 
 {
-    static void Main(string[] args)
+    static void Main(string[] args) 
     {
         Console.WriteLine(" --------------------- \nWelcome to the gladiator trials!\n-----------------------");
-        Console.WriteLine("Please choose a class !");
-        Character player = characterSelection();
-        Console.WriteLine("Time to BATTLE !");
-        Enemy enemy = new Enemy();
-        battle(player, enemy);
+        Console.WriteLine("Please choose a class!");
+        Character player = CharacterSelection();
+        Console.WriteLine("Time to Battle!");
+        Character enemy = new Character("Tyler1");
+        Battle(player, enemy);
         Console.ReadKey();
     }
-    static Character characterSelection()
+    
+    static Character CharacterSelection() 
     {
-        string name = " ";
-        int characterChoice = 0;
+        string name = "";
+        int choice = 0;
         Character player = null;
-        try
+        try 
         {
             Console.WriteLine("PRESS 1 FOR WARRIOR CLASS\nPRESS 2 FOR MAGE CLASS\n...");
-            characterChoice = Convert.ToInt32(Console.ReadLine());
+            choice = Convert.ToInt32(Console.ReadLine());
             Console.WriteLine("Awesome! Now choose a name for your gladiator!");
             name = Console.ReadLine();
-
+            if (name == "")
+            {
+                name = "Dumbass";
+            }
         }
-        catch (FormatException)
+        catch (FormatException) 
         {
             Console.WriteLine("Wrong input, try again!");
         }
 
-        if (characterChoice == 1)
+        switch (choice)
         {
-            Console.WriteLine($"From here on out you are {name} the mighty warrior!");
-            player = new Mage();
+            case 1: 
+                return new Warrior(name);
+            case 2:
+                return new Mage(name);   
+            default:
+                return new Idiot(name);                
```

Comment:
Now that the print statements such as `$"From here on out you are {name} the mighty warrior!"` are run at the instantiation of the class via their constructors, we no longer need to write them here. So this can be simplified down to:

```cs
switch (choice)
{
    case 1: 
        player = new Warrior(name);
        break;
    case 2:
        player = new Mage(name);
        break;
    default:
        player = new Idiot(name);
        break;
}

return player
```

But since we're returning player at the end anyways, we might as well just return player within the switch statements itself.

```cs
switch (choice)
{
    case 1: 
        return new Warrior(name);
    case 2:
        return new Mage(name);   
    default:
        return new Idiot(name);     
}

// The code execution will never reach below the switch statement now.
```